### ISSUE TITLE
Opiniones y sugerencias del escenario

### ISSUE THREAD

**Issue by vidimus** [SylvyaVega/INSANE#1]
Hola, @SylvyaVega 

Ayer jugué tu escenario y te he dejado unos cuantos comentarios. No sé si sabes dónde se consultan los comentarios de los escenarios de Valkyrie.

Están en esta hoja de cálculo, en la hoja "(Spoilers) Review information".

[https://docs.google.com/spreadsheets/d/1OsevlouIvR0DRYhGIS41PU9lBLB8VYgJvyj9KsHcAik/edit#gid=1035295975](url)

Ya te han dejado otro comentario, por cierto.

Hay un discord (donde están los creadores de la aplicación y un buen puñado de creadores de escenarios) y un grupo de telegram de Valkyrie (en español, de la comunidad Aruok).

**Comment by SylvyaVega** [comment]
Hola @vidimus 

Gracias por jugar mi escenario, espero que te haya gustado, y gracias también por decirme dónde puedo consultar los comentarios.

Me he metido en en la hoja "(Spoilers) Review information" del enlace que me has pasado pero no encuentro mi escenario.

**Comment by vidimus** [comment]
@SylvyaVega No sé si has podido al final ver los comentarios. Usa el buscador dentro de la hoja de cálculo. Hay cerca de 11.000 entradas, pero se guardan por orden, así que si vas a las últimas líneas, a partir de la 10.900 encontrarás los comentarios de tu escenario (10904 para ser exactos, es donde está la primera).

Sí, el escenario me ha gustado, sobre todo la historia, pero creo que tiene algunas cosas que se pueden mejorar referentes al diseño. Obviamente es tu escenario, y lo mío es una simple opinión.

------------------------------------------------------ 
- La historia está bastante bien, sobre todo el principio. El final, con tanto monstruo y con tantos puntos de vida ha sido complicado con sólo 3 investigadores.

- Primer turno perdido porque empiezan al lado de una puerta que no se abre y a 3 casillas de la siguiente (imposible llegar y abrir, no importa cuántos investigadores sean, todos pierden el turno). En total son 4 turnos para poder abrir la puerta principal y nada más. Se puede arreglar fácilmente cambiado el punto de salida de los investigadores a la casilla de la puerta principal y añadiendo alguna ficha de búsqueda al lado de la otra puerta, o en las casillas intermedias.

- En el momento en el que abres la puerta principal, el mismo problema, que se pierde el turno porque no se puede activar la ficha de arriba, ya que ningún investigador llega a ella con una acción libre

- Hay varios turnos en los que hay poco que hacer con 3 investigadores, ya que sólo se avanza siguiendo un orden completamente lineal. En cambio, en el final, habría estado bien tener más investigadores,

- El diario da mucha información, y sólo cuesta una acción. Sería mejor repartir esa información en varias pistas para recopilarla poco a poco, sin que necesite avanzar la trama principal. Eso permitiría hacer más cosas durante los turnos, evitando que algunos investigadores pasen el turno sin hacer nada.

- Hay pocos objetos en el escenario. No es que me gusten los que llenan de objetos a los investigadores, pero creo que alguno más estaría bien, sobre todo que ayudasen de cara al enfrentamiento final (y que sus respectivas fichas de Búsqueda darían acciones extra que evitarían lo que comentaba de que hay veces que los investigadores no tienen nada que hacer en un turno).

--------------------------------


El grupo de telegram para la App de Valkyrie es (tienes que buscarlo en la portada de la comunidad, se llama "App Valkyrie"):
https://telegram.me/comunidadAruok

**Comment by SylvyaVega** [comment]
@vidimus Leyendo lo que me has escrito entiendo que los comentarios no los he podido ver porque en el enlace solo aparece hasta la línea 10178. No sé si es error del enlace o mío.

Gracias por tu opinión, la verdad es que me resulta muy útil. Ya me di cuenta de algunos de esos errores cuando lo probé una vez subido a los escenarios de Valkyrie, intentaré arreglarlos.

Me apunto el grupo de telegram.

**Comment by vidimus** [comment]
@SylvyaVega 

Parece que el archivo no está actualizado. Mira en la última pestaña, la que pone "Raw Data", que ésa suele funcionar cuando la otra no se actualiza correctamente.

**Comment by SylvyaVega** [comment]
@vidimus Gracias, ya he podido ver mis comentarios.

En caso de querer subir una versión modificada del escenario, ¿os la mando por el mismo sitio por el que os pasé por primera vez el escenario?

**Comment by vidimus** [comment]
> @vidimus Gracias, ya he podido ver mis comentarios.
> 
> En caso de querer subir una versión modificada del escenario, ¿os la mando por el mismo sitio por el que os pasé por primera vez el escenario?

No, simplemente sustituye los archivos del repositorio, pero con el mismo nombre. Es importante que no cambies los nombres o no se actualizará. 

Para hacerlo, le das a subir ficheros al repositorio, arrastras a la ventana que se abre tus archivos nuevos y le das a confirmar. Si los nombres son idénticos se sobreescribirán y en unas horas el servidor de Valkyrie lo marcará como listo para ser actualizado.

Espero haberme explicado, y si tienes alguna duda, mencioname que llega un correo de aviso.

**Comment by vidimus** [comment]
@SylvyaVega Hola, ¿cómo va la actualización del escenario?

**Comment by vidimus** [comment]
¿Hola, @SylvyaVega ?

**Comment by SylvyaVega** [comment]
Hola siento no haber contestado 
Al final no creo que lo pueda actualizar
Lo siento

**Comment by vidimus** [comment]
> Hola siento no haber contestado Al final no creo que lo pueda actualizar Lo siento

Hola, @SylvyaVega 

Vaya, pues es una pena, creo que el escenario tenía bastante potencial de mejora, y aspectos muy logrados. ¿Ya no juegas al Mansiones? ¿Hay algo en lo que pueda colaborar para mantener o mejorar tu escenario?

Un saludo.
